server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -69,4 +80,16 @@ Authenticator provideAuthenticator(MetricFactory metricFactory,
             basicAuthenticationStrategy,
             jwtAuthenticationStrategy);
     }
+
+    @Provides
+    @Singleton
+    JmapRfcConfiguration provideConfiguration(PropertiesProvider propertiesProvider) throws ConfigurationException, MalformedURLException {
+        try {
+            Configuration configuration = propertiesProvider.getConfiguration("jmap");
+            return new JmapRfcConfiguration(new URL(configuration.getString("jmap-rfc-8621-base-path")));
+        } catch (FileNotFoundException e) {
+            LOGGER.warn("Could not find JMAP configuration file. JMAP server will not be enabled.");
+            throw new ConfigurationException("Property[jmap-rfc-base-path] at 'jmap.properties' is mandatory.");
```

Comment:
Let's rather use `http://localhost`as a base URL when none is specified ?

Reply:
Is it really desirable ?